Gemfile

```diff
@@ -54,7 +54,7 @@ end
 
 group :build do
   gem 'cim',                :require => false
-  gem 'mof',                :require => false, :git => 'https://github.com/puppetlabs/mof.git'
+  gem 'mof',                :require => false, :git => 'https://github.com/puppetlabs/mof.git', :branch => '2f42f1c'
```

Comment:
I always thought that bundler used `:ref` for this... are `:ref` and `:branch` interchangeable?

Should probably double check Travis / AppVeyor to ensure we pulled the right version.  Should also probably wait until we merge h the new changes to MOF repo in before we merge this.